health/health.d/backend.conf

```diff
@@ -6,7 +6,7 @@
    every: 1m
     warn: $this > 0
    delay: down 5m multiplier 1.5 max 1h
-    info: The backends subsystem is deprecated and will be removed soon. Migrate your configuration to exporting.conf.
+    info: the backends subsystem is deprecated and will be removed soon. Migrate your configuration to exporting.conf.
```

Comment:
I prefer the uppercase here, because it is the beginning of the sentence.

Reply:
i see no we follow that rule anywhere. Also there is no `.` at the end of infos.